docs/source/examples/visualization/plot_display.py

```diff
@@ -28,7 +29,7 @@
 project = create_example(fldr, "nauru")
 
 # %%
-# We grab all the links data as a Pandas dataframe so we can process it easier
+# We grab all the links data as a geopandas GeoDataFrame so we can process it easier
```

Comment:
In this example you can also add a cell showing that you can call links.plot() for a very quick visualization